src/smallUsefulFunctions.cpp

```diff
@@ -226,7 +226,7 @@ QString textToLatex(const QString &text)
 	QList<QPair<QString, QString> > replaceList;
 	// replacements for resevered characters according to
 	// http://en.wikibooks.org/wiki/LaTeX/Basics#Reserved_Characters
-	replaceList.append(QPair<QString, QString> ("\\", "\\textbackslash{}"));

```

Comment:
{} is a valid construct. Not sure why that should be changed.

Reply:
because it will be turned into \{\} by the two lines below it

Reply:
then it is probably best to move this line further down
The argument for ```\textbackslash{}``` usually is that you can add a space which is then shown or add text, which is placed directly behind.

Reply:
if you move it further down, the backslashes of escaped curly braces will be turned into `\\textbackslash`, that was the first thing I tried. I considered replacing `\textbackslash\{\}` with `\textbackslash{}` again, but that seemed a bit hacky

Reply:
I see

Reply:
I fixed the issue in the repository

Reply:
I think you accidentally put the line in the wrong place, it now escapes backslashes for the other special characters.

e.g. try to convert `\{}#$%&~_^\\{}\` to latex

Reply:
true, fixed